Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "keyvaluestore"
-version = "0.1.0"
+version = "1.0.0"
```

Comment:
I figured we could do this since it's correct, but no worries if we don't want to bother changing this.